plugins/blip/modalday/ModalDay.js

```diff
@@ -451,9 +457,19 @@ module.exports = {
     };
     _.defaults(opts, defaults);
 
+    var bgDomain = [opts.bgDomain[0]];
+    if (opts.clampTop) {
+      if (opts.bgUnits === 'mg/dL') {
+        bgDomain.push(400);
```

Comment:
Well, I think we use the Dexcom threshold as a convenient value to choose to top out at. We're not even showing CGM data in Trends right now, so it's not directly applicable. But it's what we clamp to in one-day view, so it makes sense to use the same value if clamping is on.

(In mmol/L it's not exactly 22.5, but more like 22.2, but I picked a slightly rounder number. LMK if you think there's a better value for that scale.)